lib/backup/actions/restore.go

```diff
@@ -202,3 +207,20 @@ func (sw *statWriter) Write(p []byte) (int, error) {
 	atomic.AddUint64(sw.bytesWritten, uint64(n))
 	return n, err
 }
+
+func createRestoreLock(dst string) error {
+	lockF := path.Join(dst, "restore.lock")
```

Comment:
I'd rename this file to `restore-in-progress`, since `restore.lock` is a bit misleading

Reply:
Fixed